<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>733</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;Sitka&quot;;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="autoFormatting">
       <set>QTextEdit::AutoBulletList</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_us"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFormatting">
    <property name="title">
     <string>Formatting</string>
    </property>
    <addaction name="actionItalic"/>
    <addaction name="actionBold"/>
    <addaction name="actionStrikeOut"/>
    <addaction name="actionUnderline"/>
    <addaction name="actionBacklight"/>
    <addaction name="separator"/>
    <addaction name="actionAlignLeft"/>
    <addaction name="actionAlign_enter"/>
    <addaction name="actionAlignRight"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormatting"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(26, 26, 26);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCreate"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_us"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionItalic"/>
   <addaction name="actionBold"/>
   <addaction name="actionStrikeOut"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionBacklight"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlign_enter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCreate">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>:/ico/doks/create.png</normaloff>:/ico/doks/create.png</iconset>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>:/ico/doks/open.png</normaloff>:/ico/doks/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>:/ico/doks/save.png</normaloff>:/ico/doks/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_us">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>:/ico/doks/save us.png</normaloff>:/ico/doks/save us.png</iconset>
   </property>
   <property name="text">
    <string>Save us</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>:/ico/doks/undo.png</normaloff>:/ico/doks/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>:/ico/doks/redo.png</normaloff>:/ico/doks/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>:/ico/doks/copy.png</normaloff>:/ico/doks/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>:/ico/doks/paste.png</normaloff>:/ico/doks/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>:/ico/doks/cut.png</normaloff>:/ico/doks/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>:/ico/doks/about.png</normaloff>:/ico/doks/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset theme="format-text-italic"/>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset theme="format-text-bold"/>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionStrikeOut">
   <property name="icon">
    <iconset theme="format-text-strikethrough"/>
   </property>
   <property name="text">
    <string>StrikeOut</string>
   </property>
   <property name="toolTip">
    <string>StrikeOut</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset theme="format-text-underline"/>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionBacklight">
   <property name="icon">
    <iconset theme="weather-clear"/>
   </property>
   <property name="text">
    <string>Backlight</string>
   </property>
   <property name="toolTip">
    <string>Backlight</string>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="icon">
    <iconset theme="format-justify-left"/>
   </property>
   <property name="text">
    <string>AlignLeft</string>
   </property>
  </action>
  <action name="actionAlign_enter">
   <property name="icon">
    <iconset theme="format-justify-center"/>
   </property>
   <property name="text">
    <string>Align–°enter</string>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="icon">
    <iconset theme="format-justify-right"/>
   </property>
   <property name="text">
    <string>AlignRight</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
